src/main/resources/application.properties

```diff
@@ -8,4 +8,5 @@ services.vendia.baseUrl=https://hdoprvkmp1.execute-api.us-west-2.amazonaws.com/g
 services.vendia.apiKey=${API_KEY}
 
 spring.security.oauth2.client.registration.google.clientId=${CLIENT_ID}
-spring.security.oauth2.client.registration.google.clientSecret=${CLIENT_SECRET}
\ No newline at end of file
+spring.security.oauth2.client.registration.google.clientSecret=${CLIENT_SECRET}
+spring.security.oauth2.client.registration.google.scope=openid, https://www.googleapis.com/auth/userinfo.email
```

Comment:
Good call on adding scopes, I'm sure our application isn't going to tamper with one's data in the first place, but it's better practice on our end :)

Reply:
Actually, now that we are using webflux we need to define scopes because now Spring Security is not mvc-based but webflux-based and we cannot access the SecurityContext